.github/workflows/main.yml

```diff
@@ -34,6 +34,7 @@ jobs:
           pip install sentencepiece
           pip install -e .
           if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+          python -c "import nltk; nltk.download('averaged_perceptron_tagger_eng')"
```

Comment:
Just curious why is nltk added here?

Reply:
Not adding nltk made test_tree_pipeline_cmd in test_agent_pipeline.py to fail. It only happened recently, and wasn't an issue when I submitted the visualization PR. I have no clue why, but the error message told me that I need to nltk.download so I did